styles/globals.css

```diff
@@ -6,8 +6,17 @@
   .inputIcon {
     @apply flex items-center space-x-1 hover:bg-gray-100 flex-grow justify-center p-2 rounded-xl cursor-pointer;
   }
+
   .btn {
     @apply bg-gray-200 p-3 rounded-md ring-gray-200 text-sm text-gray-800 
     hover:ring-1 focus:outline-none active:ring-gray-300 hover:shadow-md;
   }
+
+  .buttonText::after {
+    display: block;
+    content: attr(data-text) / "";
```

Comment:
I checked this out, fired up the server and still had issues, but all I needed to add was one field here to get it to work. Add the following:
`content: attr(data-text);` 

Not sure why another one is necessary, but that does the trick.